### DISCUSSION TITLE
Brickrail LEGO Train automation - Alpha release trailer

### DISCUSSION THREAD

**Discussion by Novakasa** [pybricks/support#1033] [Show and tell]
Hey everyone, yesterday I created my first alpha release of my train automation project, so I created a small trailer for fun! Since the project would not have been possible without Pybricks I wanted to share with this community as well!

[Trailer on youtube](https://www.youtube.com/watch?v=UZj41oFs1KA)

[Github page](https://github.com/Novakasa/brickrail)

[Wiki](https://github.com/Novakasa/brickrail/wiki)

**Comment by laurensvalk** [comment]
Amazing project! Pybricks users seem to like it too: https://m.facebook.com/story.php?story_fbid=pfbid02CMfzCQ5Tfqm5DHHShFhYK4Es8en34xox6nenNaZHoc8dsfxKR335x9BBVcGEmW6Zl&id=100060705685488

**Reply by Novakasa** [reply]
Nice, thanks for sharing!

I've also shared this on a few other places, and it was pretty popular on the [godot engine subreddit](https://www.reddit.com/r/godot/comments/12ob5w8/my_godotpowered_open_source_lego_train_automation/) especially

**Comment by Novakasa** [comment]
Just noticed I accidentally posted the link to the quick guide instead of the trailer 🤦 
Here's the trailer: https://www.youtube.com/watch?v=UZj41oFs1KA
